starport/templates/app/stargate/Makefile.plush

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Can you try to avoid scaffolding a makefile only for this, please?

Reply:
It is needed to Akash docker image to build the project. I think it is also useful to provide a option to allow users to build the project without install Starport

Reply:
We can use the `starport chain build` command. And if you are scaffolding a module with starport, you already have it installed. The build should be for developers. The users should manage the signed binary.

Reply:
Akash need to build a docker image, and docker image does not have starport installed. Moreover, it will use `make install` to install the node binary. https://github.com/ovrclk/cosmos-omnibus/blob/master/Dockerfile#L25

Reply:
We can run the commands into the Dockerfile. The Makefile is not necessary.

Reply:
The Makefile is used across for all projects in `cosmos-omnibus`